pkg/reconciler/taskrun/resources/apply.go

```diff
@@ -399,6 +399,22 @@ func getTaskResultReplacements(spec *v1.TaskSpec) map[string]string {
 	return stringReplacements
 }
 
+func ApplyArtifacts(spec *v1.TaskSpec) *v1.TaskSpec {
```

Comment:
docstrings for exported function and structs.

Reply:
Thank you ! I missed it!